version.props

```diff
@@ -3,6 +3,7 @@
     <VersionPrefix>2.1.0</VersionPrefix>
     <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix>
     <ExperimentalVersionSuffix>alpha1</ExperimentalVersionSuffix>
+    <ExperimentalVersionSuffix Condition="'$(ExperimentalVersionSuffix)' != '' And '$(BuildNumber)' != ''">$(ExperimentalVersionSuffix)-$(BuildNumber)</ExperimentalVersionSuffix>
```

Comment:
This will produce the following when IsFinalBuild=true: `0.4.0-alpha1-12345-final`. Can you fix it?